tensorflow_model_optimization/python/core/clustering/keras/cluster_wrapper_test.py

```diff
@@ -252,68 +254,41 @@ def assert_all_weights_associated(weights, centroid_index):
     # Weights should now be all clustered with the centroid 1
     assert_all_weights_associated(l.layer.kernel, centroid_index=1)
 
-
-  def testClusterReassociation(self):
```

Comment:
Hi @johan-gras,
Could you please elaborate on why this is a duplicated test for `testSameWeightsAreReturnedBeforeAndAfterSerialisation`?

Reply:
> with another test with the same name a few lines above (line 194).

Oh!!

> This is unrelated to testSameWeightsAreReturnedBeforeAndAfterSerialisation

Agreed.